### PR TITLE
Update local-groups.mdx

### PR DESCRIPTION
**Pull request by wowtah**
Added Europe as a group
Moved UK under Europe
Added The Netherlands Telegram group
Ordered the list A-Z

### CODE REVIEW

File: docs/community/local-groups.mdx [hunk 1]
```diff
@@ -20,6 +20,13 @@ us on [Discord](https://discord.com/invite/ktMAKGBnBs) to add your group.
 ### Alberta
 - [YYC Mesh](https://yycmesh.com/)
 
+## Europe
```
[the remark this answers is not in the document]
Comment (wowtah) [reply docs/community/local-groups.mdx:23]: sure. Not very experienced with git, let me see if I can do it.

[the remark this answers is not in the document]
Comment (wowtah) [reply docs/community/local-groups.mdx]: ok cool. I think it is okay now?

**Comment by caveman99** [comment]
> Change requested for formatting reasons. I'm also going to have an Admin review this because of the name as it could be an issue with the trademark guidelines. I'll let them make that decision though.

I think this case is fine, since it's a user group on the community page. I am not an admin, but it'd say this is fair community use.

**Comment by wowtah** [comment]
> I think it's just because it _can imply_ it's official rather than a community group because of the use of the country alongside it rather than region. 

Our whole country is about the size of Kentucky 😅. The group has 109 members at the moment.

**Comment by mc-hamster** [comment]
> > Change requested for formatting reasons. I'm also going to have an Admin review this because of the name as it could be an issue with the trademark guidelines. I'll let them make that decision though.
> 
> I think this case is fine, since it's a user group on the community page. I am not an admin, but it'd say this is fair community use.

This.

**Pull request merged**